style.css

```diff
@@ -473,3 +514,320 @@ img.arrow {
 .project1:active {
   background-color: rgba(34, 48, 210, 1);
 }
+
+.desktopnav_bar {
+  display: none;
+}
+
+#grid_container__desktop {
+  display: none;
+}
+
+/* <<<<<<<<<<<DESKTOP VERSION>>>>>>>>>>>>>>>>> */
+
+@media only screen and (min-width: 900px) {
+  .nav_bar {
```

Comment:
- [ ] Please make sure to use `768px` not `900px` as the minimum width for desktop view, [as per requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=Desktop%3A%20768px%20or%20wider).